### PR TITLE
MileStone4:  connect with external API

### PR DESCRIPTION
**Pull request by isume295**
In this milestone I:

- Connected project to external API
- Used axious to fetch and update data
- Refactored Add and Delete Function
- Added status

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @isume295 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 The project's requirements are implemented correctly ✔️ 
👍🏽 The PR is professional with a title and a description ✔️ 
👍🏽 The linters are installed and the PR passes the check ✔️ 
👍🏽 The commit messages are descriptive ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] The section of setup in your readme file conveys a piece of misleading information about the way to clone your repo, the URL should be: `https://github.com/isume295/BookStore`
Screenshot 2023-06-16 at 17 42 47


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-import store from './redux/store';
+// import { getBooks } from './redux/books/booksSlice';
 
 function App() {
+  // const { bookItem } = useSelector((store) => store.book);
+  // const dispatch = useDispatch();
+  // useEffect(() => {
+  //   dispatch(getBooks());
+  // }, []);
+
```
Comment (cynthiainga) [inline src/App.js:16]: - [OPTIONAL] It would be better to delete the commented line of code to have a clean codebase 🙃

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ 🎉 ✔️ 

Hi @isume295, 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] `axios` or `fetch`  is used ✔️ 
- [ ] `createAsyncThunk` is imported into the project ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can get the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (isume295) [reply src/App.js:16]: okay, I will delete, thank you

**Pull request merged**